.github/workflows/dotnet.yml

```
@@ -79,14 +79,14 @@
       run: dotnet restore aspnet-core/toyiyo.todo.sln
     - name: Cache SonarCloud packages
-      uses: actions/cache@v1
+      uses: actions/cache@v2
       with:
         path: ~\sonar\cache
         key: ${{ runner.os }}-sonar
         restore-keys: ${{ runner.os }}-sonar
     - name: Cache SonarCloud scanner
       id: cache-sonar-scanner
-      uses: actions/cache@v1
+      uses: actions/cache@v2
```

actionlint (1.7.4)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)